Pod/Classes/PINRemoteImageManager.m

```diff
@@ -508,6 +508,21 @@ - (NSUUID *)downloadImageWithURL:(NSURL *)url
         }
     }
     
+    if ([url.absoluteString hasPrefix:@"data:"]) {
+        NSData *data = [NSData dataWithContentsOfURL:url];
+        if (data) {
+            id object;
+            if ([url.resourceSpecifier isEqualToString:@"image/gif"]) {
```

Comment:
Hi Anton - thanks for implementing this feature!

For caching, I think it's an important question.  Using these URLs as cache keys will increase the size of the cache, as the compressed image will itself be contained in the key.  This might be OK for apps that choose to use this feature, but something to consider.

On Sep 5, 2016, at 3:17 PM, Anton Bukov <notifications@github.com> wrote:
> 
> In Pod/Classes/PINRemoteImageManager.m <https://github.com/pinterest/PINRemoteImage/pull/240#discussion_r77557602>:
> 
> > @@ -508,6 +508,21 @@ - (NSUUID *)downloadImageWithURL:(NSURL *)url
> >          }
> >      }
> >      
> > +    if ([url.absoluteString hasPrefix:@"data:"]) {
> > +        NSData *data = [NSData dataWithContentsOfURL:url];
> > +        if (data) {
> > +            id object;
> > +            if ([url.resourceSpecifier isEqualToString:@"image/gif"]) {
> My fail
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/pinterest/PINRemoteImage/pull/240/files/d4f3728f69b022e1a175f2b40b4ec48351aff5e4#r77557602>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAigA6IMEEG3SbwHzOuA0LC3zuOff6V0ks5qnJUWgaJpZM4J1GeO>.
>